jetty-ee9/jetty-ee9-webapp/src/main/java/org/eclipse/jetty/ee9/webapp/MetaInfConfiguration.java

```diff
@@ -166,7 +166,8 @@ public void findAndFilterContainerPaths(final WebAppContext context) throws Exce
         if (classPath != null)
         {
             Stream.of(classPath.split(File.pathSeparator))
-                .map(URIUtil::toURI)
+                .map(resourceFactory::newResource)
```

Comment:
This works, but isn't ideal as the stream converts strings to resources to uris then `addContainerResource()` converts them back to resources. Another subtle change that doesn't have any adverse effect but hampers readability (IMHO) is the fact that mounting happens here and not in `addContainerResource()` anymore.

This also ignores that `ResourceFactory.newResource()` may return null.